src/DataBaseTree/Views/Templates/DataGenerationConfigWindowTemplate.xaml

```diff
@@ -0,0 +1,11 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:DBManager.Application.Views.Templates"
+                    xmlns:windows="clr-namespace:DBManager.Application.ViewModels.Windows">
+
+    <DataTemplate DataType="{x:Type windows:DataGanarationConfigurationViewModel}">
```

Comment:
empty window :D ??

Reply:
I gave him this task. I will develop the contents of the window.